cmd/syncer/main.go

```diff
@@ -171,21 +182,17 @@ func startSyncers(ctx context.Context, GVRs []string, secretRef client.ObjectKey
 
 	downstreamDynamicClient, err := dynamic.NewForConfig(mgr.GetConfig())
 	if err != nil {
-		return fmt.Errorf("could not make dynamic downstream client from manager rest config: %v", err.Error())
+		return nil, fmt.Errorf("could not make dynamic downstream client from manager rest config: %v", err.Error())
 	}
 	SpecSyncer, err := spec.NewSpecSyncer(upstreamClusterConfig.Name, upstreamClusterConfig.Name, upstreamClient, downstreamDynamicClient)
 	if err != nil {
-		return fmt.Errorf("could not create new spec syncer: %v", err.Error())
+		return nil, fmt.Errorf("could not create new spec syncer: %v", err.Error())
 	}
 	logger.Info("starting syncer", "name", upstreamClusterConfig.Name)
 
-	go SpecSyncer.Start(ctx, SYNCER_THREADS)
```

Comment:
i dont think the const declaration of this is being used anywhere else in the file

Reply:
oh yeah, I have another PR coming shortly, that I've already addressed that in, so I'll leave it for that PR.

Reply:
ok sounds good